lib/src/Pos/Wallet/Web/ClientTypes/Instances.hs

```diff
@@ -157,12 +158,12 @@ instance FromMultipart CElectronCrashReport where
 
 instance HasTruncateLogPolicy CTx where
     -- Related logs are printed often.
-    -- Tx history changes increamentally, order is newest first
+    -- Tx history changes incrementally, order is newest first
     truncateLogPolicy = take 5
 
 instance HasTruncateLogPolicy CWallet where
     -- Related logs are printed seldom.
-    -- Contains constantly chaning info (balance)
+    -- Contains constantly changing info (balance)
```

Comment:
I have a feeling that I typed these on
https://im0-tub-ru.yandex.net/i?id=7a4daadd3b98690d51f7ad5e56b581f8&n=13